packages/components/src/FormatFile/FormatFile.tsx

```diff
@@ -1,29 +1,69 @@
 import React, { useState } from "react";
 import filesize from "filesize";
+import classNames from "classnames";
 import { IconNames } from "@jobber/design";
 import styles from "./FormatFile.css";
-import { Button } from "../Button";
+import { FormatFileDeleteButton } from "./FormatFileDeleteButton";
+import { sizeToDimensions } from "./sizeToDimensions";
 import { Icon } from "../Icon";
 import { Typography } from "../Typography";
 import { ProgressBar } from "../ProgressBar";
 import { FileUpload } from "../InputFile";
 
+const progressBar = (file: FileUpload) => (
+  <div className={styles.progress}>
+    <ProgressBar
+      size="small"
+      currentStep={file.progress * 100}
+      totalSteps={100}
+    />
+  </div>
+);
+
 interface FormatFileProps {
   /**
    * File upload details object. (See FileUpload type.)
    */
   readonly file: FileUpload;
 
+  /**
+   * To display as either a file row or thumbnail
+   *
+   * @default "file"
+   */
+  readonly display?: "file" | "thumbnail";
+
+  /**
+   * The base dimensions of the thumbnail
+   *
+   * @default "default"
+   */
+  readonly displaySize?: keyof typeof sizeToDimensions;
+
+  /**
+   * Function to execute when format file is clicked
+   */
+  onClick?(event: React.MouseEvent<HTMLDivElement>): void;
```

Comment:
onClick seems to only do something when `display="thumbnail` I think it should do something when it is a file row too. @chris-at-jobber thoughts?

Reply:
That makes sense, I'd expect the same behaviour from both variations